Src/Fare/RegExp.cs

```diff
@@ -634,6 +649,95 @@ private StringBuilder ToStringBuilder(StringBuilder sb)
             return sb;
         }
 
+        public string UsedChars()
+        {
+            var sb1 = new StringBuilder();
+            return this.ToAlphabetStringBuilder(sb1).ToString();
+        }
+        private StringBuilder ToAlphabetStringBuilder(StringBuilder sb)
```

Comment:
what is this thing for /doing?

Reply:
to create charset from regex. for usage with patternelement (or regexelement) to check what chars can be there in code for generate train data for ocr training.